src/zimcheck/checks.cpp

```diff
@@ -296,10 +346,9 @@ void test_articles(const zim::Archive& archive, ErrorLogger& reporter, ProgressB
                     int index = item.getIndex();
                     if (previousIndex != index)
                     {
-                        typedef kainjow::mustache::data data;
-                        data links(data::type::list);
+                        std::string links;
                         for (const auto &olink : p.second)
-                            links << data{"link", olink};
+                            links += "- " + olink + "\n";
```

Comment:
Should we escape the `olink` here ?

Reply:
This has been obsoleted by the fix of the [other comment](https://github.com/openzim/zim-tools/pull/206#discussion_r589345568). Still, in the modified code no escaping of `olink` is performed since strings are escaped during the JSON output stage.